forge/db/migrations/20220624-01-add-devicesettings.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Add DeviceSettings table
+ */
+const { DataTypes } = require('sequelize')
```

Comment:
The migration needs to create the table.

Reply:
Done